docs/compatibility.md

```diff
@@ -434,6 +434,49 @@ The plugin supports reading `uncompressed`, `snappy` and `gzip` Parquet files an
 fall back to the CPU when reading an unsupported compression format, and will error out in that
 case.
 
+## JSON
+
+The JSON format read is a very experimental feature which is expected to have some issues, so we disable 
+it by default. If you would like to test it, you can enable `spark.rapids.sql.format.json.enabled` and 
+`spark.rapids.sql.format.json.read.enabled`.
+
+Currently, the JSON reader doesn't support the column prune when reading JSON files, which is the blocker 
```

Comment:
nit: can we reword it as

```
Currently, the GPU accelerated JSON reader doesn't support column pruning, which will likely make
this difficult to use or even test.  The user must specify the full schema ...
```

Also what happens with predicate push down?  If we don't use one of those columns in a query I thought that Spark would push down to skip reading it, even if we said it was in the schema. So in the example

```
val df = spark.read.json("people.json")
df.selectExpr("SUM(age)").show()
```

would that work or not?  If not then we need to document that part too.

Reply:
It will work since this PR passes the whole dataSchema to cudf to read all columns, and then do the column pruning according to the readSchema

Reply:
Done, changed the doc